packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -938,162 +922,6 @@ export interface MetricTargetTrackingProps extends BaseTargetTrackingProps {
   readonly targetValue: number;
 }
 
-export interface BlockDevice {
```

Comment:
I'm not thrilled about this breaking change, but I couldn't find a way to cleanly deprecate `asg.BlockDevice`, as it would require the following type for `AutoScalingGroupProps`:

```ts
readonly blockDevices?: (asg.BlockDevice | ec2.BlockDevice)[];
```

Reply:
Couldn't `asg.BlockDevice` just be more-or-less a copy of `ec2.BlockDevice`?

I agree, not ideal, but at least not a breaking change.